Gamecube/gc_input/controller-WUPC.c

```diff
@@ -58,8 +58,8 @@ static button_t buttons[] = {
 	{ 13, WPAD_CLASSIC_BUTTON_PLUS, "+" },
 	{ 14, WPAD_CLASSIC_BUTTON_MINUS, "-" },
 	{ 15, WPAD_CLASSIC_BUTTON_HOME, "Home" },
-	{ 16, WUPC_EXTRA_BUTTON_RSTICK, "RS-Button" },
-	{ 17, WUPC_EXTRA_BUTTON_LSTICK, "LS-Button" },
+	{ 16, WUPC_EXTRA_BUTTON_RSTICK, "LS-Button" },

```

Comment:
Were you trying to reorder the values here? If so, I would assume WUPC_EXTRA_BUTTON_RSTICK should not map to LS-Button and vice versa